build.log

```diff
@@ -0,0 +1,1473 @@
+Sending build context to Docker daemon  8.685MB


```

Comment:
Was this committed by accident? 
I don't think we should be storing logs on github :)

Reply:
Yikes! Thanks, it was for [the ticket](https://github.com/medic/cht-conf/issues/448#issue-1048017825), so already public - but yes, was not meant to be committed, thank you! Fixed in 9373056